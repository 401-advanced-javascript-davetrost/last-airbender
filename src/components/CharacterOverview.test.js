import React from 'react';
import { shallow } from 'enzyme';
import { CharacterOverview } from './CharacterOverview';

describe('Character Overview component', () => {
  it('renders character overview', () => {
    const item = {
      name: 'Fire Nation councilwoman',
      image: 'https://vignette.wikia.nocookie.net/avatar/images/f/f9/Fire_Nation_councilwoman.png/revision/latest?cb=20121110144847',
    };

    const wrapper = shallow(<CharacterOverview name={item.name} image={item.image} handleCharacterSelect={() => true} />);
    expect(wrapper).toMatchSnapshot();
  });
});
  
