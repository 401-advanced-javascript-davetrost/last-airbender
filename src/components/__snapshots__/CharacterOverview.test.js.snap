// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Character Overview component renders character overview 1`] = `
<div
  onClick={[Function]}
>
  <h2>
    Fire Nation councilwoman
  </h2>
  <img
    alt="image of Fire Nation councilwoman"
    src="https://vignette.wikia.nocookie.net/avatar/images/f/f9/Fire_Nation_councilwoman.png/revision/latest?cb=20121110144847"
  />
</div>
`;
